Rubberduck.Core/UI/UnitTesting/UnitTestProperties.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Rubberduck.UI.UnitTesting
+{
+    class UnitTestProperties
```

Comment:
This class is suspicious... is it used anywhere? What's its purpose?

Reply:
Testing cruft I should have disposed of properly... :whoops: